src/FurCoNZ.Web/Views/Account/Orders.cshtml

```diff
@@ -23,11 +29,13 @@
                 @foreach (var ticket in order.Tickets)
                 {
                     <tr>
-                        <td>@ticket.TicketType.Name</td>
-                        <td>@(string.IsNullOrWhiteSpace(ticket.PreferredFullName) ? ticket.IdentificationFullName : ticket.PreferredFullName)</td>
-                        <td>@ticket.TicketType.Price.ToString("C")</td>
+                        <td>@Html.Raw(@s) @ticket.TicketType.Name @Html.Raw(@se)</td>
```

Comment:
This will work fine, but when we get more time this should probably be a CSS class, rather than HTML elements